showcases/patternhub/components/link-header/index.tsx

```diff
@@ -0,0 +1,16 @@
+import Link from 'next/link';
+import { DBIcon } from '../src';
+
+const LinkHeader = ({ id }) => {
+	if (id) {
+		return (
+			<Link href={`#${id}`} scroll={false}>
+				<DBIcon icon="add_link">{id}</DBIcon>
```

Comment:
```suggestion
				<DBIcon icon="add_link"></DBIcon>
```

I don't understand what for what `id` would be included in there.